modules/sdk/project-templates/project-templates-modules-ext/src/main/resources/archetype-resources/build.gradle

```diff
@@ -15,7 +15,6 @@ apply plugin: "com.liferay.osgi.ext.plugin"
 #end
 dependencies {
 	originalModule group: "com.liferay", name: "${originalModuleName}"#if (${dependencyManagementEnabled} != "true"), version: "${originalModuleVersion}"#end
-
```

Comment:
Hey @gamerson @petershin, this new line is not created by mistake and it's a really line break for `#end`. You could generate this file to test. (i was confused in here too)